geoportal/c2cgeoportal_geoportal/lib/xsd.py

```diff
@@ -81,7 +83,12 @@ def add_association_proxy_xsd(self, tb, column_property):
         association_proxy = getattr(cls, proxy)
         relationship_property = class_mapper(cls).get_property(association_proxy.target)
         target_cls = relationship_property.argument
-        query = DBSession.query(getattr(target_cls, association_proxy.value_attr))
+        if hasattr(cls, "__enumerations_order__") and column.name in cls.__enumerations_order__:
```

Comment:
IMHO we should move `if hasattr(cls, "__enumerations_order__")` in `dbreflexion._add_association_proxy` and add an `order_by` property on association proxy. It would be more understandable.
It does not really make sense here to suppose `cls` can have a `__enumerations_order__` attribute.